controllers/watcherapi_controller.go

```diff
@@ -282,6 +287,10 @@ func (r *WatcherAPIReconciler) generateServiceConfigs(
 		"MemcachedServers": memcachedInstance.GetMemcachedServerListString(),
 		"LogFile":          fmt.Sprintf("%s%s.log", watcher.WatcherLogPath, instance.Name),
 		"APIPublicPort":    fmt.Sprintf("%d", watcher.WatcherPublicPort),
+		"PrometheusHost":   string(secret.Data[PrometheusHostKey]),
+		"PrometheusPort":   string(secret.Data[PrometheusPortKey]),
+		"PrometheusTLS":    prometheusTLS,
+		"PrometheusCaCert": string(secret.Data[PrometheusCaCertKey]),
```

Comment:
so I expected to be adding these things into the DecisionEngine controller at some point in https://github.com/openstack-k8s-operators/watcher-operator/pull/44

I am a bit confused why api needs these (lets discuss today if you have some time please i wanted to catch up about /44 anyway)

Reply:
Actually, probably don't need all the config but I'd like to keep a single configuration file for all the services as it should be fully compatible.